File: pset1/cash/cash.c
#include<stdio.h>
#include<cs50.h>
#include<math.h>
int main(void)
{
    float money = 0;
    //promote to inter height
    do
    {

        money = get_float("Change owed: ");

    }

    // calculate number of change
    while (money <= 0);
    //to change and round to nearest cent
    int cents = round((money * 1000) / 10);

    //counter to count number off coin nedded
    int counter = 0;
    while (cents > 0)
    {
        if (cents >= 25)
        {
            counter++ ;
            cents -= 25;

        }
        else if (cents >= 10)
        {
            counter++ ;
            cents -= 10;

        }
        else if (cents >= 5)
        {
            counter++;
            cents -= 5;

        }
        else
        {
            counter++ ;
            cents -= 1;

        }
    }
    // print the result
    printf("%i\n", counter);


}

File: pset6/cash/cash.py
from cs50 import get_float


def main():
    # prompt user for right input
    while True:
        money = get_float("Change owed: ")
        if money > 0:
            break
    # convert to cents
    cents = round(money * 100)
    counter = 0
    # remove bigger first
    while cents > 0:
        if cents >= 25:
            counter += 1
            cents -= 25

        elif cents >= 10:
            counter += 1
            cents -= 10

        elif cents >= 5:
            counter += 1
            cents -= 5

        else:
            counter += 1
            cents -= 1
    # print result
    print(f"{counter}")


if __name__ == "__main__":
    main()